lxd/storage_utils.go

```diff
@@ -319,3 +320,34 @@ func shrinkVolumeFilesystem(s storage, volumeType int, fsType string, devPath st
 	err := shrinkFileSystem(fsType, devPath, mntpoint, byteSize)
 	return cleanupFunc, err
 }
+
+func Statvfs(path string) (*syscall.Statfs_t, error) {
+	var st syscall.Statfs_t
+
+	err := syscall.Statfs(path, &st)
+	if err != nil {
+		return nil, err
+	}
+
+	return &st, nil
+}
```

Comment:
Feels like something that'd make more sense under util?

Reply:
moved